tests/conftest.py

```diff
@@ -396,3 +401,111 @@ def log_cli_level(pytestconfig):
 
     """
     return pytestconfig.getini('log_cli_level') or 'DEBUG'
+
+
+@pytest.fixture(scope="session")
+def run_io_in_background(request):
+    """
+    Run IO during the test execution
+    """
+    if config.RUN['cli_params'].get('io_in_bg') and config.RUN['google_api_secret']:
```

Comment:
So basically when I explicitly use option `--io_in_bg`, it will be silently ignored when I don't also configure `google_api_secre`? If that is the case, make it more obvious, eg. fail pytest execution (so that no tests will be actually executed) with error message stating that `google_api_secret` is mandatory. Otherwise we are adding quite nice debugging trap ;)

Reply:
Or maybe even better, allow people to use this without the google api secret.

Reply:
removed the condition of config.RUN['google_api_secret']